core/app/models/concerns/spree/active_storage_adapter.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module Spree
+  # Adapts ActiveStorage interface to make it compliant with Solidus'
+  # Paperclip-oriented attachment support.
+  module ActiveStorageAdapter
+    extend ActiveSupport::Concern
+    include Spree::ActiveStorageAdapter::Normalization
+
+    included do
+      next if Rails.gem_version >= Gem::Version.new('6.1.0.alpha')
+
+      raise 'ActiveStorage support requires Rails >= 6.1.0. ' \
+        'Please check Spree::Config.image_attachment_module and ' \
+        'Spree::Config.taxon_attachment_module preferences values.'
```

Comment:
I think we may improve the messaging here to explain exactly what needs to be fixed in order to resolve the issue. Sorry for the shameless self-promotion, but I wrote an article a while ago about the matter that may help 😅 https://medium.com/@aldesantis/fail-better-turning-software-errors-into-documentation-764a6bb7d71f

Reply:
Thank you for the useful reference, I'll try an make the message more solution-oriented 👍